instances/ageCategory/adolescent.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/ageCategory/adolescent",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/AgeCategory",
+  "definition": "'Adolescent' categorizes the life cycle stage of an animal or human that starts right before puberty and ends right after puberty.",
```

Comment:
"adolescent" is a synonym for puberty. I would redefine this: "'Adolescent' categorizes a transitional life cycle stage of growth and development between childhood and adulthood, often described as 'puberty'". @ingvildeb: what do you think?

Reply:
Sounds good to me!